crates/pop-parachains/src/templates.rs

```diff
@@ -79,10 +79,31 @@ pub enum Template {
 		props(Provider = "Pop", Repository = "https://github.com/r0gue-io/base-parachain")
 	)]
 	Base,
```

Comment:
I think we should consider renaming `Base` here to `Standard`. Creating a base parachain sounds weird. Creating a standard parachain sounds rational. As we are still alpha, it is an opportunity to correct this.

It can still pull from the base-parachain repo, its just the UX that could be better.

Reply:
Vanilla Parachain or Default Parachain are ideal candidates as well. But Standard is something I think we can all agree on.

Reply:
Refactored to Standard as the three of us seems to agree. Default sound good to me too.